components/roode/persisted_number.cpp

```diff
@@ -0,0 +1,25 @@
+#include "persisted_number.h"
+
+namespace esphome {
+    namespace roode {
+        auto PersistedNumber::control(float newValue) -> void {
+            this->publish_state(newValue);
```

Comment:
Parent has one so I omitted because I didn't want to duplicate
https://github.com/esphome/esphome/blob/b5a0e8b2c0d016ea384f4f962e0f0e4c61a8dc85/esphome/components/number/number.cpp#L36

It's debug level, but I think logs are only relevant at the debug level anyways. Otherwise you just can observe the current state.

I'll add it though if you really want an info level as well.

Reply:
Yes because the parent has its on `DEBUG` i have this added previously as `INFO` because i need this log very often when testing things and only check the terminal. I dont use HA when developing stuff. That's a essential log line.

Reply:
K LMK what you think. Since it has nothing to do with roode or people counting now I left it as a float.